app/src/main/java/com/example/weatherapp/ui/hours/HourViewModel.kt

```diff
@@ -47,7 +47,7 @@ class HourViewModel @Inject constructor(
     @get:Bindable
     val hourValue: String?
         @RequiresApi(Build.VERSION_CODES.O)
-        get() = hourModel?.hour?.formatHour(resourceProvider, hourModel?.timeZoneOffset)
+        get() = hourModel?.timeZoneOffset?.let { hourModel?.hour?.formatHour(it) }
```

Comment:
there is a better way of doing this check as `hourModel?.` is redundant

Reply:
`hourModel` is nullable and you check if it is not null twice. Change it so the check is done only once .